HISTORY.md

```diff
@@ -1,6 +1,121 @@
 ---
 title: Release History
 ---
+# 2.3.0-rc1 (2024-04-18)
+## Highlights
+The 2.3.0-rc1 release is our first release candidate pre-release, meant to test all the new features and enhancement for the upcoming 2.3.0 release.
+
+There are two main new features:
+- The experimental cell-centric discrete spaces, as added in #1994. It allows having cells with not only properties but also active behaviors: the `CellAgent`. Its inspired by NetLogo's [patches](https://ccl.northwestern.edu/netlogo/bind/primitive/patches.html) but extend and generalize this concept further.
+- Full support for discrete event scheduling, as added in #2066. It allows scheduling events (like Agent actions) at any time, including non-integer timesteps.
+
+There are a lot of other features: The Jupyter visualisation now supports sliders, `NetworkGrid.get_neighbors()` supports a radius, `AgentSet.get()` can retrieve multiple attributes and there are now benchmarks to track Mesa performance during development.
```

Comment:
The JupyterViz has always supported sliders, the PR was to add a concise convenient wrapper.

Reply:
How would you formulate it exactly?

Reply:
@EwoutH this statement is misleading, and the PR shouldn't be merged before this is fixed.

Reply:
Could you offer a suggestion?

Reply:
"The Jupyter visualisation now supports easier way to specify sliders" is short and not specific enough, but better than the previous one.